thruster_interface/scripts/thruster_interface_node.py

```diff
@@ -48,7 +55,17 @@ def output_to_zero(self):
                 pwm_msg.positive_width_us.append(neutral_pulse_width)
             self.pub_pwm.publish(pwm_msg)
 
+    def heartbeat_callback(self, msg):
+        self.prev_heartbeat = msg.stamp
+
     def callback(self, msg):
+        if (rospy.get_rostime() - self.prev_heartbeat).to_sec() > 1.0:
```

Comment:
Should we not then also receive a heartbeat from allocator, if this is something we fear? If allocator dies but joystick interface is healthy, then the last input from the allocator will be sent to the thrusters regardless of joystick input.